torch/ao/quantization/observer.py

```diff
@@ -628,6 +637,11 @@ def __init__(
         factory_kwargs=None,
         eps=torch.finfo(torch.float32).eps,
     ) -> None:
+        if not is_per_channel(qscheme):
+            raise NotImplementedError(
+                "PerChannelMinMaxObserver's qscheme only support \
+                    torch.per_channel_symmetric and torch.per_channel_affine."
```

Comment:
we have a `torch.per_channel_affine_float_qparams` as well: https://github.com/pytorch/pytorch/blob/master/c10/core/QScheme.h#L43

Reply:
added.